graphql_compiler/query_pagination/parameter_generator.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+
+HydratedParameters = namedtuple(
+    'HydratedParameters',
+    (
+        'next_page_parameters',     # dict, parameters with which to execute the next page
+                                    # parameterized query.
+        'continuation_parameters',  # dict, parameters with which to execute the parameterized
+                                    # continuation query.
+    ),
+)
+
+
+def hydrate_parameters_of_parameterized_query(
+    schema_graph, statistics, parameterized_pagination_queries, num_pages
+):
+    """Generate parameters combining the user's parameters and parameters for paging.
```

Comment:
why is this combining called hydration? Can you point me to a definition of the word that applies in this context?

Reply:
My line of thinking was that I could easily call this function `generate_parameters` and be done with it. However, what we're doing here is we're assigning values to the parameterized queries i.e. creating parameter values for filters which do not have one. 
Replaced `hydrate` with `generate`.

Reply:
the word hydrate still appears throughout the PR

Reply:
Fixed.